presto-native-execution/README.md

```diff
@@ -26,10 +26,19 @@ available inside `presto/presto-native-execution/scripts`.
 * CentOS Stream 9: `setup-centos.sh`
 * Ubuntu: `setup-ubuntu.sh`
 
-Create a directory say `dependencies` and invoke one of these scripts from
-this folder. All the dependencies are installed in the system default location eg: `/usr/local`.
-To change the installation location specify a path using the `INSTALL_PREFIX` environment variable.
-For example, change the location if the default location cannot be written to by the user running the setup script.
+The above setup scripts use the `DEPENDENCY_DIR` environment variable to set the
+location to download and build packages. This defaults to `deps-download` in the current
+working directory.
+
+Use `INSTALL_PREFIX` to set the install directory of the packages. This defaults to
+`deps-install` in the current working directory on macOS and to the default install
+location (eg. `/usr/local`) on linux.
+Using the default install location `/usr/local` on macOS is discouraged since this
+location is used by certain Homebrew versions.
+
+Manually add the `INSTALL_PREFIX` value in the IDE or bash environment,
```

Comment:
```suggestion
Manually add the `INSTALL_PREFIX` value in the IDE or bash environment so subsequent Velox builds can use the installed packages.
```
Suggest revising this to describe in one sentence what you want the user to do, then separate out the example as a second sentence. I've tried doing this here, let me know if my suggestion makes sense.